hedera-node/hedera-schedule-service-impl/src/main/java/com/hedera/node/app/service/schedule/impl/serdes/MonoSchedulingStateAdapterCodec.java

```diff
@@ -17,33 +17,32 @@
 package com.hedera.node.app.service.schedule.impl.serdes;
 
 import com.hedera.node.app.service.mono.state.merkle.MerkleScheduledTransactionsState;
-import com.swirlds.common.io.streams.SerializableDataInputStream;
-import com.swirlds.common.io.streams.SerializableDataOutputStream;
+import com.hedera.pbj.runtime.Codec;
+import com.hedera.pbj.runtime.io.ReadableSequentialData;
+import com.hedera.pbj.runtime.io.WritableSequentialData;
 import edu.umd.cs.findbugs.annotations.NonNull;
 import java.io.IOException;
 
 public class MonoSchedulingStateAdapterCodec implements Codec<MerkleScheduledTransactionsState> {
     @NonNull
     @Override
     public MerkleScheduledTransactionsState parse(final @NonNull ReadableSequentialData input) throws IOException {
-        if (input instanceof SerializableDataInputStream in) {
-            final var context = new MerkleScheduledTransactionsState();
-            context.deserialize(in, MerkleScheduledTransactionsState.CURRENT_VERSION);
-            return context;
-        } else {
-            throw new IllegalArgumentException("Expected a SerializableDataInputStream");
-        }
+        return new MerkleScheduledTransactionsState(input.readLong());
```

Comment:
This read and the write below look pretty different than what was originally here. That's fine as long as we didn't miss anything

Reply:
The MerkleScheduledTransactionsState does the same in its serialize.  It's not perfect, but would require rewriting the original object in MonoService to make the serialization take PBJ DataIO to fix more cleanly.  As functionality is moved out of MonoService, this should definitely be revisited.